src/bg/help/guides/getting-started.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Първи стъпки
+lang: bg-BG
+---
+
+# Първи стъпки
+
+## Инсталация
+**Paperback** понастоящем _не е_ записана в **TestFlight** и _изисква_ да използвате **AltStore**, за да инсталирате приложението на iPhone или iPad. В бъдеще TestFlight ще бъде използван, когато главният разработчик придобие [Apple Developer акаунт](https://developer.apple.com/programs/).
+
+::: tip Видеоупътване
+Ако предпочитате видеоупътване, Artesians има прекрасно видео на темата. Ще го намерите [тук](https://www.youtube.com/watch?v=n1KRwsxNiWY) за Уиндоус и [тук](https://www.youtube.com/watch?v=CjPjsF4yJ0M) за macOS.
```

Comment:
Can I ask a question? I find it strange to translate `Windows`. 
Moreover in [Microsoft website](https://www.microsoft.com/bg-bg/windows) the word used is `Windows`. 
Could you confirm the translation is intended?

Reply:
It is fine both ways, although I suppose the original version is better. I agree with this change.

Reply:
Solved in most recent commit